### PR TITLE
Script to download all member images from Firebase Storage

### PR DESCRIPTION
**Pull request by riyajaggi**
### Summary 

This pull request implements a script `download_images.ts` to download all member images from the `images` bucket in IDOL's Firebase Storage and saves them with the same name (i.e. _netid.jpg_). It does not affect any existing functionality.

### Test Plan 

- [x] Run the script to ensure all images are downloaded.
image

### Breaking Changes 

None

### CODE REVIEW

**Review by henry-li-06 — COMMENTED** [review]
So our backend already has an endpoint `/allMemberImages` that does not require authentication and gives a list of download URLs for each image in the Firebase storage. I think this script can just fetch that endpoint and use the download URLs to download the images, somewhat similar to what was done for the member JSONs in #50. That way this script doesn't need to connect to our firebase instance.

File: backend/data/download-images.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+import fetch from 'node-fetch';
+import { ProfileImage } from '../src/DataTypes';
+
+const fs = require('fs-extra');
```
Comment (SamChou19815) [inline backend/data/download-images.ts:6]: Change this to use import and `yarn workspace backend add fs-extra`

**Review by SamChou19815 — CHANGES_REQUESTED** [review]
Almost good, just one change.

File: backend/data/download-images.ts (hunk 1 above)
Comment (riyajaggi) [reply backend/data/download-images.ts]: done!

Comment (henry-li-06) [reply backend/data/download-images.ts]: image
There's still another linting error that needs to be fixed

**Review by SamChou19815 — APPROVED** [review]

**Pull request merged**